src/daria/corrections/shape/curvature.py

```diff
@@ -53,12 +55,12 @@ def __init__(self, config: Optional[dict] = None, **kwargs) -> None:
 
         Arguments:
             kwargs (Optional keyword arguments):
-                config (dict, optional): config dictionary; default is None
+                config (dict, str, Path): config dictionary; default is None. Either this
+                            or the image_source must be provided.
                 image_source (Union[Path, np.ndarray]): image source that either can
```

Comment:
When we change `config_source` to `config`, one should also change `image_source` to `img` (or `image)`. But I would guess we use the first more consistently. What do you think? In some sense I again do not like it, as one will have to update all scripts. But on the other hand, one would now anyways have to change this line in every script.

Reply:
What is your sense here? I understood that we do not have to update scripts due to the config source change.. Should I change here as well? I guess it only affects the setup routines which people are not doing that often.

Reply:
Should be fine here.